lms/static/scripts/frontend_apps/components/dashboard/DashboardActivityFilters.tsx

```diff
@@ -126,6 +126,24 @@ function StudentOption({
   );
 }
 
+/**
+ * A badge to display an amount of items in a filter, as long as it is greater
+ * than 0.
+ * If the value is greater than the page size, we display 99+ instead, since
+ * the actual value is not known
+ */
+function CountBadge({ count }: { count?: number }) {
```

Comment:
I think you probably want to revise this so that the caller can indicate whether the count is an accurate value or a lower bound (so should be rendered as `{count}+`). This way we can easily vary the number of items per page for different requests.

Reply:
Yeah, makes sense. It should not be up to this component to know what is the max value to display.